packages/shared/routes/dashboard/governance/views/GovernanceEventDetails.svelte

```diff
@@ -59,27 +90,24 @@
     <Text type="p" smaller overrideColor classes="text-gray-800">{localize('actions.back')}</Text>
 </div>
 
-<div class="w-full h-full grid grid-cols-3 gap-x-4 min-h-0">
-    <DashboardPane classes="w-full h-full p-6 col-span-2 flex flex-col">
-        <Text
-            type="p"
-            classes="mr-auto uppercase px-2 py-1 mb-2 text-blue-500 bg-blue-100 rounded-lg"
-            smaller
-            bold
-            overrideColor>{event?.status?.status}</Text
-        >
+<div class="w-full h-full grid grid-cols-3 grid-rows-2 gap-4 min-h-0">
+    <DashboardPane classes="w-full h-full p-6 col-span-2 row-span-2 flex flex-col">
+        <div class="flex flex-start items-center mb-2">
+            <Text type="p" classes="px-2 py-1 text-blue-500 bg-blue-100 rounded-lg" smaller bold overrideColor
+                >{localize(`views.governance.events.status.${event?.status?.status}`)}</Text
+            >
+            <Icon icon="info-filled" classes="ml-2 text-gray-400" />
+        </div>
         <Text type="h2" classes="mb-4">{event?.information?.name}</Text>
         <Text type="p" classes="mb-2">{event?.information?.additionalInfo}</Text>
         <Text type="p" classes="mb-2">{event?.information?.payload?.questions[0]?.text}</Text>
         <Text type="p" classes="mb-6">{event?.information?.payload?.questions[0]?.additionalInfo}</Text>
-        {#each event?.information?.payload?.questions[0]?.answers as answer}
-            <Button
```

Comment:
Let's leave this as a button, since this was a specific request to be able to disable it based on the event.status